pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -90,7 +90,7 @@ stdenv.mkDerivation {
     # Binutils, and Apple's "cctools"; "bintools" as an attempt to find an
     # unused middle-ground name that evokes both.
     inherit bintools;
-    inherit libc nativeTools nativeLibc nativePrefix isGNU isClang default_cxx_stdlib_compile;
+    inherit libc nativeTools nativeLibc nativePrefix isGNU isClang isLibCxx default_cxx_stdlib_compile;
```

Comment:
What about introducing libcxx similar to libc we have now? I think we want to add that in the future anyway to get rid of the `default_cxx_stdlib_compile` hack.

Reply:
Is there documentation anywhere of what exactly the `libc` we have now *is*?